examples/rllab/green_wave.py

```diff
@@ -6,7 +6,7 @@
 from rllab.envs.gym_env import GymEnv
 
 from flow.core.vehicles import Vehicles
-from flow.core.traffic_lights import TrafficLights
+from flow.core.params import TrafficLights
```

Comment:
We should renamed this TrafficLightsParams then

Reply:
done!